docker/docker-compose.mysql-nossl-server-ssession-proxy.yml

```diff
@@ -0,0 +1,190 @@
+            # Mount the whole ./.acrakeys directory to be able generate keys and
+            # place them in services' subdirectories
+            - ./.acrakeys:/keys
+        # Please specify ACRA_CLIENT_ID environment variable, otherwise run with
+        # default 'testclientid' client id
+        command: >-
+            --client_id=${ACRA_CLIENT_ID:-testclientid}
+            --acraserver
+            --output=/keys/acraserver
+            --output_public=/keys/acraproxy
+    # Create keys:
+    # - ./.acrakeys/acraproxy/${ACRA_CLIENT_ID}
+    # - ./.acrakeys/acraserver/${ACRA_CLIENT_ID}.pub
+    acra_genkeys_proxy:
+        image: "cossacklabs/acra_genkeys:${ACRA_DOCKER_IMAGE_TAG:-latest}"
+        network_mode: "none"
+        environment:
+            ACRA_MASTER_KEY: ${ACRA_MASTER_KEY:-UHZ3VUNNeTJ0SEFhbWVjNkt4eDdVYkc2WnNpUTlYa0E=}
+        volumes:
+            - ./.acrakeys:/keys
+        command: >-
+            --client_id=${ACRA_CLIENT_ID:-testclientid}
+            --acraproxy
+            --output=/keys/acraproxy
+            --output_public=/keys/acraserver
+    # Create keys:
+    # - ./.acrakeys/acraserver/${ACRA_CLIENT_ID}_storage
+    # - ./.acrakeys/acrawriter/${ACRA_CLIENT_ID}_storage.pub
+    acra_genkeys_writer:
+        image: "cossacklabs/acra_genkeys:${ACRA_DOCKER_IMAGE_TAG:-latest}"
+        network_mode: "none"
+        environment:
+            ACRA_MASTER_KEY: ${ACRA_MASTER_KEY:-UHZ3VUNNeTJ0SEFhbWVjNkt4eDdVYkc2WnNpUTlYa0E=}
+        volumes:
+            - ./.acrakeys:/keys
+        command: >-
+            --client_id=${ACRA_CLIENT_ID:-testclientid}
+            --storage
+            --output=/keys/acraserver
+            --output_public=/keys/acrawriter
+
+    # Create file with accounts for HTTP access and key for decrypt it
+    # - ./.acrakeys/acraserver/httpauth.accounts
+    # - ./.acrakeys/acraserver/auth_key
+    acra_genauth:
+        image: "cossacklabs/acra_genauth:${ACRA_DOCKER_IMAGE_TAG:-latest}"
+        network_mode: "none"
+        environment:
+            ACRA_MASTER_KEY: ${ACRA_MASTER_KEY:-UHZ3VUNNeTJ0SEFhbWVjNkt4eDdVYkc2WnNpUTlYa0E=}
+        volumes:
+            - ./.acrakeys:/keys
+        command: >-
+            --set
+            --user=test
+            --password=test
+            --file=/keys/acraserver/httpauth.accounts
+            --keys_dir=/keys/acraserver/
+
+    # MySQL container
+    mysql:
+        image: mysql
+        # INSECURE!!! You MUST define your own DB name and credentials
+        environment:
+            MYSQL_ONETIME_PASSWORD: ${MYSQL_ONETIME_PASSWORD:-urLK7YJPndDWsnJC}
+            MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:-hy53uBLmBuihUK29}
+            MYSQL_DATABASE: ${MYSQL_DATABASE:-test}
+            MYSQL_USER: ${MYSQL_USER:-test}
+            MYSQL_PASSWORD: ${MYSQL_PASSWORD:-test}
+        # Open the port outside for writer
+        ports:
+            - "3306:3306"
+        # We use interconnectionernal 'acraserver-mysql' network for acraserver and
```

Comment:
`interconnectionernal` -> `interconnected` / `inner`?

Reply:
Thanks! Will fix it in next commit.